README.md

```diff
@@ -0,0 +1,754 @@
+# UCAN Delegation Specification
+## Version 1.0.0-rc.1
+
+## Editors
+
+- [Brooklyn Zelenka], [Witchcraft Software]
+
+## Authors
+
+- [Brooklyn Zelenka], [Witchcraft Software]
+- [Daniel Holmgren], [Bluesky]
+- [Irakli Gozalishvili], [Protocol Labs]
+- [Philipp Krüger], [number zero]
+
+## Dependencies
+
+- [UCAN]
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119].
+
+# Abstract
+
+This specification describes the representation and semantics for delegating attenuated authority between principals. UCAN Delegation provides a cryptographically verifiable container, batched capabilities, hierarchical authority, and a minimal syntatically-driven policy langauge.
+
+# Introduction
+
+UCAN Delegation is a delegable certificate capability system with runtime-extensibility, ad hoc conditions, cacheability, and focused on ease of use and interoperability. Delegations act as a proofs for [UCAN Invocation]s.
+
+Delegation provides a way to "transfer authority without transferring cryptographic keys". As an authorization system, it is more interested in "what can be done" than a list of "who can do what". For more on how Delegation fits into UCAN, please refer to the [high level spec][UCAN].
+
+# [UCAN Envelope] Configuration
+
+## Type Tag
+
+The UCAN envelope tag for UCAN Delegation MUST be set to `ucan/dlg@1.0.0-rc.1`.
+ 
+## Delegation Payload
+
+The Delegation payload MUST describe the authorization claims, who is involved, and its validity period.
+
+| Field   | Type                                      | Required | Description                                                 |
+|---------|-------------------------------------------|----------|-------------------------------------------------------------|
+| `iss`   | `DID`                                     | Yes      | Issuer DID (sender)                                         |
+| `aud`   | `DID`                                     | Yes      | Audience DID (receiver)                                     |
+| `sub`   | `DID`                                     | Yes      | Principal that the chain is about (the [Subject])           |
```

Comment:
We discussed powerline concepts in https://github.com/ucan-wg/delegation/issues/11 and I believe came to an agreement that setting `sub: null` would imply re-delegation of everything that had been delegated to the issuer however spec does not seem to mention it and in fact table contradicts it. Can we please make sure that powerlines  are covered ?

```suggestion
| `sub`   | `DID \| null`                             | Yes      | Principal that the chain is about (the [Subject])           |
```

Reply:
Fixed ✅